openjdk.test.jlm/doc/jlm.md

```diff
@@ -0,0 +1,113 @@
+java.lang.management Test Suite 
+===============================
+
+Contents
+---------- 
+1. What does it test?
+2. How is it implemented?
+3. What setup is required?
+4. How do I run the tests?
+5. How do I know whether it worked or failed?
+6. Is there anything else I should know?
+7. Are there any references?
+
+###What does it test? 
+
+This testsuite tests the java.lang.management API, new from Java 5.0, but including the **LockInfo** and **MonitorInfo** classes introduced with Java 6.0. This API provides functions which can be used to inspect the JVM's state and modify its properties via a set of managed beans, known as Platform MBeans.
+
+Each running JVM has a set of these Platform MBeans. The java.lang.management API can be used with the MBeans for the local JVM, or it can be used with the MBeans of remote JVMs. Therefore a Java application on one machine can use java.lang.management functions to monitor and manage Java applications
+running on remote machines, as long as the remote applications have been started with security options which allow them to be remotely monitored.
```

Comment:
In the bullets
First item alone is seen in grey color and with different indentation where as second item onward its black color.